test/features/paths_spec.rb

```diff
@@ -0,0 +1,8 @@
+require './test/test_helper'
+
+class Paths < ActionDispatch::IntegrationTest
+  def test_can_visit_the_root_path_without_authentication
+    visit '/'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.